test/lib/runtime/survey_action_test.exs

```diff
@@ -0,0 +1,91 @@
+defmodule Ask.Runtime.SurveyActionTest do
+  use Ask.ModelCase
+  use Ask.MockTime
+  use Ask.TestHelpers
+  alias Ask.Runtime.SurveyAction
+  alias Ask.Survey
+
+  describe "start/1" do
+
+    test "generates a panel survey if generates_panel_survey is ON" do
+      survey = panel_survey_generator_survey()
+
+      {:ok, %{survey: survey}} = SurveyAction.start(survey)
+
+      assert survey.panel_survey_id
+    end
+
+    test "names the generated panel survey with the survey name" do
```

Comment:
Yes, we're testing the same behavior twice, but on different modules. I guess I'm missing something, but I cant' see the reason to discard one of these tests. Could you please enlighten me? Wich of them would you discard?

Reply:
Because I don't want to lose test coverage, I'm keeping both. Please let's discuss this further in the future, I think it's a very interesting topic. I'd like to look for documentation about the best way of covering the model and the controller without duplicating the logic.

Thanks for the heads up, @anaPerezGhiglia! I really appreciate it!